scripts/spi_flash/board_defs.py

```diff
@@ -159,6 +167,7 @@
     'btt-octopus-f446-v1.1': BOARD_DEFS['btt-octopus-f446-v1'],
     'btt-octopus-pro-f429-v1.0': BOARD_DEFS['btt-octopus-f429-v1'],
     'btt-octopus-pro-f446-v1.0': BOARD_DEFS['btt-octopus-f446-v1'],
+    'btt-octopus-pro-h723-v1.1': BOARD_DEFS['btt-octopus-pro-h723-v1'],
```

Comment:
@locki-cz -- I think all you need to do is first rebase your PR onto the current master and just add an alias for the Octopus Pro.  I did look at the schematic for it [https://raw.githubusercontent.com/bigtreetech/BIGTREETECH-OCTOPUS-Pro/master/Hardware/BIGTREETECH%20Octopus%20Pro%20V1.1-sch.pdf](https://raw.githubusercontent.com/bigtreetech/BIGTREETECH-OCTOPUS-Pro/master/Hardware/BIGTREETECH%20Octopus%20Pro%20V1.1-sch.pdf) and it appears to be the same pinout and MCU as the SKR3 for the SDIO connections.

So you should be able to change this line to

```
'btt-octopus-pro-h723-v1.1': BOARD_DEFS['btt-skr-3-h723']
```

after you rebase your PR.

You could also add an alias for the v1 entry as well:
```
'btt-octopus-pro-h723-v1': BOARD_DEFS['btt-skr-3-h723']
```
however, I don't see where BTT explicitly mentioned a v1 with the H723 -- I only see it on the v1.1 schematic.  But, it's possible they have a v1 with that MCU in addition to the v1.1.  Since most of the STM32 family has interchangeable parts, BTT has been making use of that lately -- probably because of supply-chain issues.